src/graphql/blocks.ts

```diff
@@ -65,6 +66,33 @@ async function tsToBlockNum(network, ts) {
   return to.number;
 }
 
+async function tsToBlockNum(network, ts) {
+  const explorer_base_url = networks[network].explorer.apiUrl ?? networks[network].explorer.url;
```

Comment:
We shouldn't mix casing for vars but only use camelCase
```suggestion
  const explorerBaseUrl = networks[network].explorer.apiUrl ?? networks[network].explorer.url;
```

Reply:
:see_no_evil:  old habits from Ruby and Python, thanks for catching that